web/src/frontend/src/digisos/skjema/oppsummering/index.tsx

```diff
@@ -103,8 +110,14 @@ class OppsummeringView extends React.Component<Props, {}> {
 		if (this.props.visBekreftMangler) {
 			classNames += " skjema-oppsummering__bekreft___feil";
 		}
+
+		let restStatusUpdated = restStatus;
```

Comment:
Hvorfor ny variabel for kopiering av enum-verdi?

Reply:
Litt usikker, men tror at hvis jeg forrandrer restStatus så er jeg redd for å mutere redux store. Så for å være helt sikker så klona jeg den